config/locales/single_page_subscriptions.yml

```diff
@@ -0,0 +1,14 @@
+en:
+  single_page_subscriptions:
+    title: The way you subscribe to emails from GOV.UK is changing
+    account_description_html: |
+      <p class="govuk-body">You need a new GOV.UK account to get these emails.</p>
+      <p class="govuk-body govuk-!-font-weight-bold">GOV.UK accounts are new</p>
+      <p class="govuk-body">They are a first step towards creating a single account where you can access all your government services.</p>
+      <p class="govuk-body">GOV.UK accounts will eventually replace all other governemnt accounts.</p>
```

Comment:
In the Figma, it says "You need a GOV.UK account", not "You need a **new** GOV.UK account".

Should the "GOV.UK accounts are new" line be a heading?  If not we're kind of emulating a heading here with bold text, which doesn't feel great for accessibility.